scripts/shared.js

```diff
@@ -13,7 +13,8 @@ const colors = require('colors')
 const pluginPathFile = path.join(__dirname, '..', '.pluginpath')
 
 /**
- * @returns {boolean} whether file exists
+ * @param fullPath
+ * @returns {Promise<boolean>} wether the file exists
```

Comment:
This is a good catch. should say "whether"

Reply:
haha oh yes :)